lib/PuppeteerSharp.Tests/Puppeteer/PuppeteerConnectTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Puppeteer
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class PuppeteerConnectTests : PuppeteerBaseTest
```

Comment:
`PuppeteerBaseTest` creates an instance of `Browser` so the `originalBrowser ` should not be created

Reply:
That's true, I'll update my tests!